src/components/Header/TopNav/LogoLink.js

```diff
@@ -31,10 +32,16 @@ const LogoLink = ({ path = '/' }) => {
   const service = isSpecialityPage ? getService(path) : serviceTitle
   const speciality = isSpecialityPage ? getSpeciality(path) : null
 
-  let originalFillColor = isSpecialityPage
-    ? logoColors[service][speciality]
-    : logoColors['default']
-  const [fillColor, setFillColor] = useState(originalFillColor)
+  let metaFillColor = themeHelper.colors.white
```

Comment:
the fill colour isn't really a meta value, I'd say we can just call it fillColor. This is a minor one I know but just wanted to point it out.